src/cdk/v2/destinations/loops/procWorkflow.yaml

```
@@ -0,0 +1,80 @@
+
+  - name: buildResponse
+    template: |
+      const response = $.defaultRequestConfig();
+      response.body.JSON = $.context.payload;
+      response.endpoint = $.context.endpoint;
+      response.headers = {
+        "authorization": "Bearer " + .destination.Config.apiKey,
+        "content-type": "application/json"
+      };
+      response.method = $.context.method;
+      response
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)